lib/appsignal/hooks/sidekiq.rb

```diff
@@ -41,8 +72,10 @@ def call(_worker, item, _queue)
 
         Appsignal.instrument "perform_job.sidekiq" do
           begin
+            job_status = :processed # TODO: is same as success?
```

Comment:
There's a todo :)

Reply:
yes, turns out `processed` is not the same as `success`. `processed` is any job, failed or succeeded.